test/testdata/deployednettemplates/recipes/txnsync/topology.json

```diff
@@ -0,0 +1,44 @@
+{ "Hosts":
+  [
+    {
+      "Name": "R1",
+      "Template": "AWS-US-EAST-2-Large"
+    },
+    {
+      "Name": "R2",
+      "Template": "AWS-US-WEST-1-Large"
+    },
+    {
+      "Name": "R3",
+      "Template": "AWS-EU-WEST-2-Large"
+    },
+    {
+      "Name": "R4",
+      "Template": "AWS-EU-CENTRAL-1-Large"
+    },
+    {
+      "Name": "N1",
+      "Template": "AWS-US-EAST-2-Large"
```

Comment:
This topology doesn't seem correct. Refer to the txync deployment document.

Reply:
Do you mean the names should be updated?  Or the Machine sizes?  Or Both?

Reply:
Nope, this is correct, I'm mistaken